superset-frontend/src/dashboard/components/PropertiesModal.tsx

```diff
@@ -38,33 +38,14 @@ import ColorSchemeControlWrapper from 'src/dashboard/components/ColorSchemeContr
 import { getClientErrorObject } from '../../utils/getClientErrorObject';
 import withToasts from '../../messageToasts/enhancers/withToasts';
 import '../stylesheets/buttons.less';
+import Owner from '../../types/Owner';
```

Comment:
Webpack has a "src" alias, which we can use for these imports rather than relative path. In this case `src/types/Owner`

We're trying to update these wherever we see them, while we're changing other things. It makes refactors (moving files around) easier.